test.py

```diff
@@ -1,4 +1,8 @@
 from test1 import func
 
```

Comment:
Unnecessary import. Remove unused import statements.

```suggestion
Remove `from test1 import func`
```